views/index.handlebars

```diff
@@ -1,3 +1,5 @@
 <script src="static/js/index.js"></script>
 
+{{> navbar}}
```

Comment:
The navbar should be put in `main.handlebars` so that it can persist across the other pages. Putting it in `index.handlebars` will mean we would have to copy this line to all the other views we create but adding it to the main layout file will keep it in one place and reduce repetition.